aerogear-windows-otp/Totp.cs

```diff
@@ -6,18 +6,37 @@ namespace AeroGear.OTP
 {
     public class Totp
     {
+        public enum Digits
```

Comment:
@sindilevich agreed, but then at least add all the options to the Digits enum that are also in DIGITS_POWER, wdyt?